contracts/multiply/Exchange.sol

```diff
@@ -92,13 +92,20 @@ contract Exchange {
     return balance;
   }
 
-  function _collectFee(address asset, uint256 fromAmount) internal returns (uint256) {
+  function _collectFeeTokenToDai(address asset, uint256 fromAmount) internal returns (uint256) {
```

Comment:
since "asset" is allways DAI_ADDRESS in both invocation maybe it will be clearer to remove asset parameter

Reply:
Fixed